### PR TITLE
Bug 1212905 - Make scroll snapping optional. r=gasolin,gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3e966a71d4d8d4f0926995b9d8eeb1046e755c4c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6cb448afb61e6383b05a5f73be8df625603e36bc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=69a752b421c02f0f0dccc71dffd61c0a0635edbc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=805da34eb364867fcc7bd112f3faa5612351d914

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7e96e253fe1fc4df33b4934a2f5e44996a19717b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=45ed323b1c5a1c68035a6641c9686c0f46217e22

File: apps/homescreen/js/app.js [hunk 1]
```diff
@@ -1,20 +1,9 @@
 /* global MozActivity, HomeMetadata, Datastore, Pages, LazyLoader, FirstRun */
+/* global Settings */
```
Comment (gmarty) [inline apps/homescreen/js/app.js]:
```js
 /* global MozActivity, HomeMetadata, Datastore, Pages, LazyLoader, FirstRun
           Settings */
```

Works too and is more readable IMO (ultra minor nit).

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a1e2faefc4f98fc6c46d208f8e997b191852e1b2

**Pull request merged**